2048.py

```diff
@@ -295,12 +295,12 @@ def draw_button_bar():
 
     pg.display.update()
 
-def board_create(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p):
+def board_create(locations):
     print("                                        ")
-    print("| {} | {} | {} | {} |".format(a, b, c, d))
-    print("| {} | {} | {} | {} |".format(e, f, g, h))
-    print("| {} | {} | {} | {} |".format(i, j, k, l))
-    print("| {} | {} | {} | {} |".format(m, n, o, p))
+    print("| {} | {} | {} | {} |".format(*locations[0:4]))
```

Comment:
This may look complicate, but `0:4`, turns this into a list with 4 elements.  Then the `*` turns the list into parameters to format()

Reply:
huh, nice